kbase-extension/static/kbase/js/widgets/function_output/kbaseUploadFile.js

```diff
@@ -129,7 +124,7 @@
                     error.statusText +
                     '\
 </div>';
-                if (typeof uploader.callback == 'function') {
```

Comment:
`uploader` is not defined -- guessed it probably mean `ul`, which is the abbrev used elsewhere